{"name": "Тест: неуспешная подписка", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x562acd8f44e3 <unknown>\n#1 0x562acd623c76 <unknown>\n#2 0x562acd65fc96 <unknown>\n#3 0x562acd65fdc1 <unknown>\n#4 0x562acd6997f4 <unknown>\n#5 0x562acd67f03d <unknown>\n#6 0x562acd69730e <unknown>\n#7 0x562acd67ede3 <unknown>\n#8 0x562acd6542dd <unknown>\n#9 0x562acd65534e <unknown>\n#10 0x562acd8b43e4 <unknown>\n#11 0x562acd8b83d7 <unknown>\n#12 0x562acd8c2b20 <unknown>\n#13 0x562acd8b9023 <unknown>\n#14 0x562acd8871aa <unknown>\n#15 0x562acd8dd6b8 <unknown>\n#16 0x562acd8dd847 <unknown>\n#17 0x562acd8ed243 <unknown>\n#18 0x7fa2a5409b43 <unknown>", "trace": "browser = <selenium.webdriver.remote.webdriver.WebDriver (session=\"29b6b794757254f8a009431372102fa6\")>\n\n    @allure.title(\"Тест: неуспешная подписка\")\n    @allure.tag('UI')\n    def test_wrong_mail_signup(browser):\n        browser.get(urls.SIGNUP_PAGE_URL)\n        with allure.step('Поиск сообщения при некорректном входе'):\n            try:\n>               UserSignupPage(browser).wrong_mail_signup(users.get_wrong_email())\n\ntests/tests_ui/test_ui.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages/newsletter_signup_page.py:26: in wrong_mail_signup\n    WebDriverWait(self.driver, 5).until(EC\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"29b6b794757254f8a009431372102fa6\")>, method = <function visibility_of_element_located.<locals>._predicate at 0x7fc9a3a633a0>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x562acd8f44e3 <unknown>\nE       #1 0x562acd623c76 <unknown>\nE       #2 0x562acd65fc96 <unknown>\nE       #3 0x562acd65fdc1 <unknown>\nE       #4 0x562acd6997f4 <unknown>\nE       #5 0x562acd67f03d <unknown>\nE       #6 0x562acd69730e <unknown>\nE       #7 0x562acd67ede3 <unknown>\nE       #8 0x562acd6542dd <unknown>\nE       #9 0x562acd65534e <unknown>\nE       #10 0x562acd8b43e4 <unknown>\nE       #11 0x562acd8b83d7 <unknown>\nE       #12 0x562acd8c2b20 <unknown>\nE       #13 0x562acd8b9023 <unknown>\nE       #14 0x562acd8871aa <unknown>\nE       #15 0x562acd8dd6b8 <unknown>\nE       #16 0x562acd8dd847 <unknown>\nE       #17 0x562acd8ed243 <unknown>\nE       #18 0x7fa2a5409b43 <unknown>\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException"}, "steps": [{"name": "Поиск сообщения при некорректном входе", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x562acd8f44e3 <unknown>\n#1 0x562acd623c76 <unknown>\n#2 0x562acd65fc96 <unknown>\n#3 0x562acd65fdc1 <unknown>\n#4 0x562acd6997f4 <unknown>\n#5 0x562acd67f03d <unknown>\n#6 0x562acd69730e <unknown>\n#7 0x562acd67ede3 <unknown>\n#8 0x562acd6542dd <unknown>\n#9 0x562acd65534e <unknown>\n#10 0x562acd8b43e4 <unknown>\n#11 0x562acd8b83d7 <unknown>\n#12 0x562acd8c2b20 <unknown>\n#13 0x562acd8b9023 <unknown>\n#14 0x562acd8871aa <unknown>\n#15 0x562acd8dd6b8 <unknown>\n#16 0x562acd8dd847 <unknown>\n#17 0x562acd8ed243 <unknown>\n#18 0x7fa2a5409b43 <unknown>\n\n", "trace": "  File \"/home/alisapokormlyak/Desktop/otus_project/project/tests/tests_ui/test_ui.py\", line 42, in test_wrong_mail_signup\n    UserSignupPage(browser).wrong_mail_signup(users.get_wrong_email())\n  File \"/home/alisapokormlyak/Desktop/otus_project/project/pages/newsletter_signup_page.py\", line 26, in wrong_mail_signup\n    WebDriverWait(self.driver, 5).until(EC\n  File \"/home/alisapokormlyak/Desktop/otus_project/project/venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1691657742503, "stop": 1691657747973}], "start": 1691657740881, "stop": 1691657747986, "uuid": "0dde63cd-ec95-4626-9cf2-6a46f08ea145", "historyId": "c41cc7f293cc12ba6c3df49030c27b6a", "testCaseId": "c41cc7f293cc12ba6c3df49030c27b6a", "fullName": "tests.tests_ui.test_ui#test_wrong_mail_signup", "labels": [{"name": "tag", "value": "UI"}, {"name": "parentSuite", "value": "tests.tests_ui"}, {"name": "suite", "value": "test_ui"}, {"name": "host", "value": "QA07"}, {"name": "thread", "value": "114568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_ui.test_ui"}]}